src/App.js

```diff
@@ -1,14 +1,50 @@
 import './App.css';
+import { useEffect, useLayoutEffect } from 'react';
+import axios from "axios"
 import { Home } from './Pages/Home/Home';
 import { Shop } from './Pages/Shop/Shop';
 import { ProductPage } from "./Pages/ProductPage/ProductPage"
 import { Login } from "./Pages/AuthenticationPages/Login"
 import { Signup } from "./Pages/AuthenticationPages/Signup"
+import { Wishlist } from './Pages/Wishlist/Wishlist';
```

Comment:
Suggestion: can import all pages files from `index.js` of pages folder

Reply:
Done